corehq/apps/app_manager/xform_builder.py

```diff
@@ -225,7 +226,9 @@ def _append_to_data(self, name, groups=None):
             node = self._data
         node.append(etree.Element(name))
 
-    def _append_to_translation(self, name, label, group=None, choices=None, label_safe=False, **params):
+    def _append_to_translation(self, name, labels, group=None, choices=None, label_safe=False, **params):
+        if not isinstance(labels, dict):
+            labels = {'en': labels}
```

Comment:
Hard-coded `'en'`s make me nervous. Do you think it would be feasible/desirable to do something like add a `langs` property to `XFormBuilder` that defaults to `['en']`?

Reply:
It corresponds to the default language used in the [form template](https://github.com/dimagi/commcare-hq/blob/3b73f6f274494d89c29042736d314ac5e38b3b3b/corehq/apps/app_manager/xform_builder.py#L44-L44).  

I do think it would be doable to make this dynamic, though I'm not sure what the benefit would be.  I'm not sure that is yet needed to be configurable.  This is used for tests, and I don't think tests should bother specifying a language unless testing translations.  I could see an argument for requiring you to specify up-front what languages you're going to use if you _are_ providing translations though.  I don't know, not sure it's worth it.

Reply:
Not being dynamic seems fine, I was thinking more along the lines of replacing this with a constant.